src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.operation;
+
+import java.util.Map;
```

Comment:
Unused imports should be removed to keep the code clean. If the `Map` import is not used elsewhere in the file, it should be removed.